compiler/rustc_ast_passes/messages.ftl

```diff
@@ -79,6 +79,9 @@ ast_passes_extern_types_cannot = `type`s inside `extern` blocks cannot have {$de
     .suggestion = remove the {$remove_descr}
     .label = `extern` block begins here
 
+ast_passes_extern_without_abi = extern declarations without an explicit ABI are disallowed
+    .help = in previous editions, a default ABI was inferred, but this is no longer the case, so you must specify one explicitly
+
```

Comment:
We can probably tighten up this wording a bit and add a suggestion with a placeholder.

Reply:
If you would check the box allowing maintainers to push to your branch, I have a suggested commit for this and the other suggestion here.

Reply:
I have checked the box. What are your suggestion words?

Reply:
@rustbot ready